### ISSUE TITLE
Details of the Blazor dialog modal will overflow if the text is too long.

### ISSUE THREAD

**Issue by gdlcf88** [abpframework/abp#7991]
When the detail is too long, it overflows:

![K6SM___V6SRKCQ`LG_OZ_IN](https://user-images.githubusercontent.com/30018771/110320428-3aa92c00-804b-11eb-9c5c-50174c7bc9cd.png)

if I add this code to `main.css`:
```css
.modal .modal-dialog .modal-content {
    word-wrap: break-word;
    word-break: normal;
}
```

the problem will be solved:

![8UWGH(@YCNAX_V{5~{5GGJJ](https://user-images.githubusercontent.com/30018771/110320849-dfc40480-804b-11eb-8ea3-6a041138e2ab.png)

**Comment by ilkayilknur** [comment]
Thanks for the report @gdlcf88.
@stsrki Can you look into that, please? Is this something you need to change in blazorise, or should we resolve it on the ABP side?

**Comment by stsrki** [comment]
I believe this is fixed with the latest 0.9.3 version where I moved modal-backdrop outside of `modal` div element.

**Comment by ilkayilknur** [comment]
Thanks, the fix will be released with 4.3.
